.markdownlint.yaml

```diff
@@ -27,6 +27,9 @@ MD029:
 # Ignore inline HTML. We need this in certain places
 MD033: false
 
+# Ignore bare URLs, as this does not work with HUGO shortcodes
+MD034: false
+
```

Comment:
This is necessary because otherwise CI checks fail with the `roadmap=https://...` shortcode parameter values introduced in this CR.

MD034 is the "No bare URLs" rule, which tries to enforce user-friendly links instead of bare URLs.